pkgs/misc/emulators/dolphin-emu/default.nix

```diff
@@ -1,13 +1,17 @@
-{ stdenv, pkgconfig, cmake, bluez, ffmpeg, libao, mesa, gtk2, glib
-, gettext, libpthreadstubs, libXrandr, libXext, readline
-, openal, libXdmcp, portaudio, SDL, wxGTK30, fetchurl
+{ stdenv, gcc5, pkgconfig, cmake, bluez, ffmpeg, libao, mesa, gtk2, glib
+, gettext, git, libpthreadstubs, libXrandr, libXext, readline
```

Comment:
/cc @MP2E on this one, I copied this over from `master.nix` so I figured it was there for a reason...